src/helix-ui/elements/HXCheckboxElement.less

```diff
@@ -24,20 +25,22 @@
   vertical-align: middle;
   width: 100%;
   z-index: 10;
+}
 
-  &:hover {
-    background-color: @cyan-50;
-    color: @cyan-500;
-  }
+/* icons */
+#hxMinus,
+#hxTick {
+  height: 1em;
+  line-height: 1;
+  width: 1em;
+}
 
-  /* icons */
-  #hxMinus,
-  #hxTick {
-    display: none;
-    height: 1em;
-    line-height: 1;
-    width: 1em;
-  }
+#hxMinus {
+  display: var(--hxMinus-display);
+}
+
+#hxTick {
+  display: var(--hxTick-display);
 }
 
 #hxNativeControl:checked:not(:indeterminate) ~ #hxCustomControl #hxTick {
```

Comment:
You can probably remove everything from line 46 onward. This is old CSS that no longer applies to the new Shadow DOM markup.